CATALOG.md

```diff
@@ -296,6 +296,19 @@ Result Type|normative
 Suggested Remediation|Ensure that your container has passed the Red Hat Container Certification Program (CCP).
 Best Practice Reference|https://connect.redhat.com/sites/default/files/2022-05/Cloud%20Native%20Network%20Function%20Requirements%201-3.pdf Section 5.3.7
 Exception Process|There is no documented exception process for this.
+#### container-is-certified-digest
+
+Property|Description
+---|---
+Test Case Name|container-is-certified-digest
+Test Case Label|affiliated-certification-container-is-certified-digest
+Unique ID|http://test-network-function.com/testcases/affiliated-certification/container-is-certified-digest
+Version|v1.0.0
+Description|http://test-network-function.com/testcases/affiliated-certification/container-is-certified-digest Tests whether container images listed in the configuration file have passed the Red Hat Container Certification Program by there digest(CCP).
```

Comment:
Are the containers listed in the configuration file tested with this test suite? Because my impression is that here the autodiscovered containers are tested instead. See my big comment.
Also, typo: _there_ digest -> _their_ digest

Reply:
its the The containers that are autodiscovered

Reply:
i updated the PR,
regarding to the list im printing it

Reply:
i fixed now